templates/index.mustache

```diff
@@ -0,0 +1,70 @@
+<?php
+
+function populate_site_info() {
+
+	$site_name = $_SERVER['HTTP_HOST'];
+	$db_select = select( [], array( 'site_url' => $site_name ), 'sites', 1 );
+
+	return $db_select[0];
+
+}
+
+function select( $columns = array(), $where = array(), $table_name = 'services', $limit = null ) {
+
+	$db_path = '/opt/easyengine/ee.sqlite';
```

Comment:
can't we read this path from env variables?

Reply:
No. `EE_CONF_ROOT` is a constant defined in EE core. It is not present as an env variable which can be used from `index.php` of `admin-tools`.

Reply:
I mean we can keep it in Env variable and then use it when needed.

Reply:
Okay :+1: